packages/pwa-kit-extension-sdk/src/react/utils/helpers.ts

```diff
@@ -33,24 +33,21 @@ export const applyHOCs = <T extends React.ComponentType<any>>(Component: T, hocs
  */
 export function cacheMethodResult(instance: any, methodName: string, cacheProperty: string) {
     const originalMethod = instance[methodName]
+    if (typeof originalMethod !== 'function') return
```

Comment:
A bit of refactoring to flatten the nested code. Here we immediately return if the method does not exist.